g2p/constants.py

```diff
@@ -0,0 +1,43 @@
+"""
+This file is for constants that can be initialized without any (expensive) dependencies.
+"""
+import os
+from abc import ABC, abstractmethod
+
+DISTANCE_METRICS = [
+    "weighted_feature_edit_distance",
+    "hamming_feature_edit_distance",
+    "feature_edit_distance",
+    "dolgo_prime_distance",
+    "fast_levenshtein_distance",
+    "levenshtein_distance",
+]
+
+LANGS_DIR = os.path.join(os.path.dirname(__file__), "mappings", "langs")
+LANGS_FILE_NAME = "langs.json.gz"
+NETWORK_FILE_NAME = "network.json.gz"
+
+
+class BaseTransducer(ABC):
```

Comment:
I mean strictly speaking these are not *constants*, so you could decide if it's worth the trouble to put them in another separate module called `g2p.types` or `g2p.abc`.

Reply:
That's a good point. I didn't like the abuse of the name, `g2p.types` would be a lot clearer.

Reply:
Refactored now.